SWLOR.Game.Server/Feature/LootTableDefinition/DantooineLootTableDefinition.cs

```diff
@@ -0,0 +1,507 @@
+                .AddItem("lth_high", 5)
+                .AddItem("lth_imperfect", 10)
+                .AddItem("fiberp_high", 5)
+                .AddItem("fiberp_imperfect", 10);
+
+            _builder.Create("DANTOOINE_ENRAGED_KATH_RARES")
+                .IsRare()
+                .AddItem("ruby", 99, 1, true)
+                .AddItem("map_70", 2, 1, true)
+                .AddItem("map_71", 2, 1, true)
+                .AddItem("map_72", 2, 1, true)
+                .AddItem("map_73", 2, 1, true)
+                .AddItem("map_74", 2, 1, true)
+                .AddItem("map_75", 2, 1, true)
+                .AddItem("map_76", 2, 1, true)
+                .AddItem("map_77", 2, 1, true)
+                .AddItem("map_78", 2, 1, true)
+                .AddItem("emerald", 1, 1, true)
+                .AddItem("red_shell_shard", 1, 1, true)
+                .AddItem("whit_shell_shard", 1, 1, true)
+                .AddItem("grn_shell_shard", 1, 1, true)
+                .AddItem("yell_shell_shard", 1, 1, true);
+        }
+
+        private void VoritorLizard()
+        {
+            _builder.Create("DANTOOINE_VORITOR_LIZARD")
+                .AddItem("wild_innards", 10)
+                .AddItem("wild_leg", 5)
+                .AddItem("tomato", 1)
+                .AddItem("cultured_butter", 1);
+
+            _builder.Create("DANTOOINE_VORITOR_LIZARD_RARES")
+                .IsRare()
+                .AddItem("ruby", 99, 1, true)
+                .AddItem("map_70", 2, 1, true)
+                .AddItem("map_71", 2, 1, true)
+                .AddItem("map_72", 2, 1, true)
+                .AddItem("map_73", 2, 1, true)
+                .AddItem("map_74", 2, 1, true)
+                .AddItem("map_75", 2, 1, true)
+                .AddItem("map_76", 2, 1, true)
+                .AddItem("map_77", 2, 1, true)
+                .AddItem("map_78", 2, 1, true)
+                .AddItem("emerald", 1, 1, true);
+        }
+
+        private void VoritorBoss()
+        {
+            _builder.Create("DANTOOINE_VORITOR_BOSS")
+                .AddItem("lth_high", 10)
+                .AddItem("lth_imperfect", 5)
+                .AddItem("fiberp_high", 10)
+                .AddItem("fiberp_imperfect", 5);
+
+            _builder.Create("DANTOOINE_VORITOR_BOSS_RARES")
+                .IsRare()
+                .AddItem("ruby", 99, 1, true)
+                .AddItem("map_70", 2, 1, true)
+                .AddItem("map_71", 2, 1, true)
+                .AddItem("map_72", 2, 1, true)
+                .AddItem("map_73", 2, 1, true)
+                .AddItem("map_74", 2, 1, true)
+                .AddItem("map_75", 2, 1, true)
+                .AddItem("map_76", 2, 1, true)
+                .AddItem("map_77", 2, 1, true)
+                .AddItem("map_78", 2, 1, true)
+                .AddItem("emerald", 1, 1, true);
+
+            _builder.Create("DANTOOINE_VORITOR_BOSS_RECIPES")
+                .AddItem("recipe_chigswd", 10)
```

Comment:
This is the second or third monster using the Chirodactyl loot tables. How many of these NPCs are there? Are they as difficult as the Chiro?